riscv_isac/main.py

```diff
@@ -30,6 +31,18 @@ def cli(verbose):
         help="Instruction trace file to be analyzed"
     )
 
+@click.option(
+        '--header-file','-h',
+        type=click.Path(resolve_path=True,readable=True,exists=True),
+        help="YAML macro file to include"
+    )
+@click.option(

```

Comment:
What are these CGF macros? What do they do?

Reply:
Please refer to the [comment](https://github.com/riscv-software-src/riscv-isac/pull/80#issuecomment-1860541754). If you want me to change the naming format for these files/commands, please let me know. 
This allows you to add a custom macro file(.yaml based) that can be used to define your personal variables that are repeatedly used in the coverpoints. So, this is similar to `encoding.h` (riscv-arch-test).

Reply:
This looks reasonable. This and all the other features/variables added in this PR will need to be documented in the respective files/sections in `docs/source/` folder. For example all variable additions go in `docs/source/cgf.rst` with some examples similar to how the existing documentation describes the `op_comb`/`csr_comb` coverpoints.

Reply:
@pawks , should I start working on the documentation, if you are okay with the implementation of other features?